src/test/kotlin/coverosR3z/timerecording/SeeTimeEntriesBDD.kt

```diff
@@ -30,30 +35,43 @@ class SeeTimeEntriesBDD {
         `then I see all of them`(entries, dbEntries)
     }
 
+    @Test
+    fun `I can't see time entries that haven't been made`(){
+        val (tru, entries) = generateSomeEntriesPreDatabase()
+
+        val dbEntries = tru.getEntriesForEmployeeOnDate(DEFAULT_EMPLOYEE, A_RANDOM_DAY_IN_JUNE_2020)
+
+        Assert.assertThrows(AssertionError::class.java) { `then I see all of them`(entries, dbEntries) }
```

Comment:
Probably could use a simpler assertion like
`assertEquals(listOf<TimeEntry>(), dbEntries)`

instead of asserting a thrown exception from this

Reply:
Done and done